### PR TITLE
Make the studio compile

### PR DESCRIPTION
**Pull request by fitzchak**

### CODE REVIEW

File: src/Raven.Server/Web/System/AdminMetricsHandler.cs [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+using System.Threading.Tasks;
+using Raven.Database.Util;
+using Raven.Server.Routing;
+using Sparrow.Json;
+
+namespace Raven.Server.Web.System
+{
+    public class AdminMetricsHandler : RequestHandler
+    {
+        [RavenAction("/admin/rootMetrics", "GET")]
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/AdminMetricsHandler.cs]: Can we be more vigilant when it comes to endpoints strings? e.g. /admin/metrics/stats

Comment (fitzchak) [inline src/Raven.Server/Web/System/AdminMetricsHandler.cs]: Feel free to change this, I just moved it away from AdminDatabasesHandler which isn't the right place for this action

Comment (ppekrol) [inline src/Raven.Server/Web/System/AdminMetricsHandler.cs]: Sure. http://issues.hibernatingrhinos.com/issue/RavenDB-4492

File: src/Raven.Server/Web/System/AdminResourcesStudioTasksHandler.cs [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+using Raven.Server.ServerWide.Context;
+using Sparrow.Json;
+using Sparrow.Json.Parsing;
+
+namespace Raven.Server.Web.System
+{
+    public class AdminResourcesStudioTasksHandler : RequestHandler
+    {
+        [RavenAction("/admin/*/toggle-disable", "POST", "/admin/{resourceType:databases|fs|cs|ts}/toggle-disable?name={resourceName:string|multiple}&isDisabled={isDisabled:bool}")]
+        public Task PostToggleDisableDatabases()
```
Comment (ppekrol) [inline src/Raven.Server/Web/System/AdminResourcesStudioTasksHandler.cs]: What do you think about http://issues.hibernatingrhinos.com/issue/RavenDB-4489 ?

Comment (ayende) [inline src/Raven.Server/Web/System/AdminResourcesStudioTasksHandler.cs]: That is where we'll be going, yes.

**Pull request merged**